File: Exercices/Java/exo-java/src/exo9.java
import java.util.Scanner;
public class exo9 {

	public static void main(String[] args) {
		
		String chaine1;
		String chaine2;
		String conc;
		
		Scanner chaine = new Scanner(System.in);
        System.out.format("Saisir la premiere chaine : ");
        chaine1 = chaine.next();
 
        System.out.format("Saisir la deuxieme chaine : ");
        chaine2 = chaine.next();
        
        conc = chaine1 + chaine2;
 
 
        System.out.format("Apres concatenation : %s\n", conc);
        chaine.close();
	}

}

File: Exercices/Langage C/exo9.c
#include <stdio.h>
#include <string.h>

int main()
{
  char str1[10], str2[10];

  printf("Entrez la premiere chaine: ");
  gets(str1);
  printf("Entrez la deuxieme chaine: ");
  gets(str2);

  strcat(str1, str2);

  printf("Apres la concatenation = %s\n", str1);

  return 0;
}
